pkg/types/schema/schema.ipldsch

```diff
@@ -0,0 +1,8 @@
+
+type Metadata struct {
+	# Previous metadata.
+	PreviousID optional Link_Metadata
+	Payload    Bytes
```

Comment:
I think we also need providers to sign the metadata. Pando would then need to verify the signature to assert that the metadata is indeed produced by the provider